packages/shorebird_cli/lib/src/artifact_builder.dart

```diff
@@ -46,7 +46,7 @@ class ArtifactBuilder {
     String? target,
     Iterable<Arch>? targetPlatforms,
     List<String> args = const [],
-    String? encodedPatchSigningPublicKey,
+    String? encodedPublicKey,
```

Comment:
Don't forget to update the doc comment above to explain this. When/how is it used, how is it encoded (although maybe changing the name to `base64PublicKey` or something might remove the need for that), etc

Reply:
I tried to come up with a good comment, but I had a hard time figuring out one. I guess just renaming, like you suggested, is already good